### DISCUSSION TITLE
Hide languages

### DISCUSSION THREAD

**Discussion by devasconceloss** [anuraghazra/github-readme-stats#1683] [Q&A] [answered]
I can't hide jupyter notebook at top languages card

**Comment by rickstaa** [comment, marked as answer]
Hey @Thiagovasc Thanks for your question. You can hide that language using the `hide` flag (i.e. `&hide=Jupyter%20Notebook`):

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=Thiagovasc&hide=Jupyter%20Notebook)](https://github.com/Thiagovasc/github-readme-stats)

**Reply by devasconceloss** [reply]
Thanks for the answer buddy
